CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# v1.6.0:
+
+### Features
+- [Feature #694](https://github.com/google/osv-scanner/pull/694)
+  Add support for NuGet lock files version 2 by @giovanni-bozzano
+
+- [Feature #655](https://github.com/google/osv-scanner/pull/655)
+  Scan and report dependency groups (e.g. "dev dependencies") for vulnerabilities
+
+- [Feature #702](https://github.com/google/osv-scanner/pull/702)
+  Created an option to skip/disable upload to code scanning by @another-rex in 
```

Comment:
Unfinished sentence?

Reply:
stripped the credits properly this time.